openedx/features/course_experience/utils.py

```diff
@@ -265,10 +266,12 @@ def reset_deadlines_banner_should_display(course_key, request):
     is_course_staff = bool(
         request.user and course_overview and has_access(request.user, 'staff', course_overview, course_overview.id)
     )
-    if is_self_paced and (not is_course_staff) and (not course_end_date or timezone.now() < course_end_date):
-        if (CourseEnrollment.objects.filter(
-            course=course_overview, user=request.user, mode=CourseMode.VERIFIED
-        ).exists()):
+    if is_self_paced and (is_course_staff) and (not course_end_date or timezone.now() < course_end_date):
```

Comment:
nit: can get rid of the parentheses around is_course_staff

Also that seems like a big changing going from something for learners to only for staff, is that intended?

Reply:
no i fixed it